test/service/test_manifest.py

```diff
@@ -184,6 +197,11 @@ def test_manifest(self):
                         self.assertEqual(200, response.status_code, 'Unable to download manifest')
                         self._assert_tsv(expected, response)
 
+            # The following assertion fails due to ManifestService attempt to use a cached manifest rather a
```

Comment:
This comment is not accurate. As you can see in https://travis-ci.org/github/DataBiosphere/azul/builds/669322139#L6920, the build does not fail at the assertion, it fails in the line above the assertion.